starter-code/lodash.html

```diff
@@ -5,6 +5,7 @@
     <script src="https://cdnjs.cloudflare.com/ajax/libs/lodash.js/4.16.4/lodash.js"></script>
     <script src="./lib/abbeyRoad.js"></script>
     <script src="./lib/harryPotter.js"></script>
+    <script src="./lib/lodash.js"></script>
```

Comment:
this is actually trying to load a non-existant file 😉 
Lodash is already loaded before